docs-web/installation/installation-with-manifests.md

```diff
@@ -209,8 +209,17 @@ $ kubectl get pods --namespace=nginx-ingress
     ```
     $ kubectl delete namespace nginx-ingress
     ```
-2. Delete the ClusterRole and ClusterRoleBinding created in that step:
```

Comment:
ClusterRole and ClusterRoleBinding were not created in the previous step at all. I suggest removal.